### PR TITLE
Add trebmuh to create-translation-issues.sh

### PR DESCRIPTION
**Pull request by jujudusud**
**Short description of changes**

Add @trebmuh as an app translator.

**Context: Fixes an issue?**

#3146 

**Does this change need documentation? What needs to be documented and how?**

No.

**Status of this Pull Request**

Ready to be merged.

**What is missing until this pull request can be merged?**

Nothing.

## Checklist

-  [X] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [X] I tested my code and it does what I want
-  [X] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#source-code-consistency) 
-  [X] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [X] I've filled all the content above

### CODE REVIEW

**Comment by trebmuh** [comment]
Mmm, that was it, thank you.

File: tools/create-translation-issues.sh [hunk 1]
```diff
@@ -71,7 +71,7 @@ declare -A TRANSLATORS_BY_LANG
 # App translators:
 TRANSLATORS_BY_LANG[app_de_DE]="rolamos"
 TRANSLATORS_BY_LANG[app_es_ES]="ignotus666"
-TRANSLATORS_BY_LANG[app_fr_FR]="jujudusud"
+TRANSLATORS_BY_LANG[app_fr_FR]="jujudusud, trebmuh"
```
Comment (ann0see) [inline tools/create-translation-issues.sh:74]:
```suggestion
TRANSLATORS_BY_LANG[app_fr_FR]="jujudusud,trebmuh"
```
Just to be safe

**Review by pljones — APPROVED** [review]

**Comment by pljones** [comment]
@ann0see, can we get this in for 3.10.0 so @trebmuh is in the script as soon as possible?

**Comment by ann0see** [comment]
If it is rebased or squashed, yes.

**Review by ann0see — APPROVED** [review]
Didn't test it but looks sensible

**Review by ignotus666 — APPROVED** [review]

**Comment by trebmuh** [comment]
Thanks for that, people. Very much appreciated.

**Pull request merged**